src/api/java/com/minecolonies/api/colony/permissions/OldRank.java

```diff
@@ -0,0 +1,30 @@
+package com.minecolonies.api.colony.permissions;
+
+/**
+ * Ranks within a colony.
+ */
+public enum OldRank
```

Comment:
Do we really need this? Couldn't this be just a utility class with constant objects for owner/officer etc?

Reply:
I'd keep it for 1.16 and remove it in 1.17